### PR TITLE
Update 4

### PR DESCRIPTION
**Pull request by homeofdonuts**
pr

### CODE REVIEW

File: a.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Just another test
 Update
 Update 2
-Update 3
\ No newline at end of file
+Update 3
+Update 4
```
Comment (test12aa) [inline a.md:5]: Fix this

File: a.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 Just another test
```
Comment (test12aa) [inline a.md:1]: Ignore this

File: a.md (hunk 1 above)
Comment (homeofdonuts) [reply a.md:5]: Done.

File: a.md (hunk 2 above)
Comment (homeofdonuts) [reply a.md]: Done.

**Pull request closed without merging**